lib/console/ovs_utils.pm

```diff
@@ -0,0 +1,43 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Basic functions for testing openvswitch
+# Maintainer: Anna Minou <anna.minou@suse.de>
+#
+package console::ovs_utils;
+
+use base Exporter;
+use Exporter;
+
+use base "consoletest";
+use testapi;
+# use utils;
+use strict;
+use warnings;
+
+our @EXPORT = qw(add_bridge ping_check);
+
+sub add_bridge {
+    my $id = shift;
+    assert_script_run("ovs-vsctl add-br br-ipsec");
+    assert_script_run("ip addr add 192.0.0.$id/24 dev br-ipsec");
+    assert_script_run("ip link set br-ipsec up");
+}
+
+sub ping_check {
+    my $server_ip = shift;
+    my $client_ip = shift;
```

Comment:
I would also add `$client_vpn` as parameter so we know where it came from:
```suggestion
my ($server_ip, $client_ip, $client_vpn) = @_;
```

Reply:
> I would also add `$client_vpn` as parameter so we know where it came from:

Changes are done ;)